### ISSUE TITLE
[Suggestion] - Celebrating Christmas South American way in MAS, More Things in Festivities

### ISSUE THREAD

**Issue by Emilio3409** [Monika-After-Story/MonikaModDev#10304]
Something I really appreciate in MAS is that you can celebrate different festivities with Monika, like her birthday, Halloween, and of course Christmas, but something I don't like in MAS is that Monika says Christmas is on the 25th after waking up, and I don't have a real problem with that and mean I guess in many countries it is, but in southamerica we celebrate Christmas on Christmas Eve or "nochebuena", all day doing Christmas activities and wait until is 12:00 p.m. sing some Christmas carols, be loud, hug, and open the gifts! I would really love to do this with Monika, I imagine her counting down until is 12:00, and then celebrating.

Or for example 31th again at 12:00 p.m. in southamerica and spain most families eat the twelve grapes to celebrate the new year.
I guess Monika After Story was never this interactive, that's why I would love it to have more things in festivities, not only about south american traditions but also about more special dialogues, or more things to do in festivities like birthdays and Halloween, or even traditions of other countries, like I don't know, China, Russia, Italy. 

This is not important, it doesn't affect the game too much and probably not ALL that people that play the game is from southamerica or other different traditions countries, even I would say we Southamericans aren't much more than the 10%, so even if you don't add anything about the south american traditions or others I would really love to see more things in festivities (For example, as another suggestion, I would like to see what are "Monika's 10 things she exects for the next year" on 31th) and I really appreciate the efforts and all the love you guys put in this game, it's okay if you don't add this, thank you for everything!

**Comment by firebeetleLobster** [comment]
I don't think it is possible for MAS to include festivals or customs of other traditions. You mentioned several countries, but there are more. If you include all their festivals into the game, then Monika will be celebrating different festivals all day long. However, if you just turn a blind eye to countries or areas that are not influential enough, then you will upset the players there. I am from China, but I am not religious. In fact, I do not want to celebrate any festival, even those in my own culture. But I think it's reasonable to make a compromise if most of the other players enjoy celebrating Christmas with Monika. After all, trying to make everyone satisfied will finally make no one satisfied.

**Comment by firebeetleLobster** [comment]
If you really want to include customs from your own culture into MAS, then maybe creating your custom branch is a better choice. It seems that there is no prevention of releasing your own branch of MAS. (Inform me if I got it wrong.)

**Comment by firebeetleLobster** [comment]
If you really want to include customs from your own culture into MAS, then maybe creating a sub-mod is a better choice.

**Comment by Emilio3409** [comment]
What I had in mind is some kind of conversation in which Monika ask you where are you from or how you celebrate different festivities or if you don't, so everyone could have their own experience and could share a special moment with Monika in the best way, obviously only including the most popular ways so adding it makes an actual difference. It's true a Submod could be the best option, but my knowledge in Python and programation in general is very poor and I don't think there is any Submodder with the motivation, time nor ability to do it.

**Comment by Emilio3409** [comment]
That's why I suggested it, because it would be really cool to see it on the game and in some strange way it inspired me the day I wrote this suggestion.
